### DISCUSSION TITLE
sdl2: Update to the head branch

### DISCUSSION THREAD

**Discussion by FtZPetruska** [Homebrew/discussions#3953] [Everyday usage] [answered]
With SDL3's development having started on the `main` branch, and SDL2 maintenance being moved to `SDL2` branch, what should the `head` branch be?

I had [this patch](https://github.com/Homebrew/homebrew-core/commit/bfafb771b140c8d64711fc6ea77c3ec8e082d459) ready to update it to the `SDL2` branch as it would have made the most sense to me, unfortunately audit complains about the branch:

```
% brew audit --strict --online sdl2
sdl2:
  * HEAD: Use `branch: "main"` to specify the default branch
Error: 1 problem in 1 formula detected
```

Should the PR with the change be open regardless? Are there options to treat `SDL2` as the default branch? Or would it be preferred to keep `main` as the default?

Thank you in advance!

**Comment by SMillerDev** [comment, marked as answer]
If the next version will be sdl 3 then head should track that development
